modules/default/weather/providers/openweathermap.js

```diff
@@ -434,6 +434,7 @@ WeatherProvider.register("openweathermap", {
 		} else if (this.firstEvent && this.firstEvent.location) {
 			params += `q=${this.firstEvent.location}`;
 		} else {
+			// TODO hide doesnt exist!
```

Comment:
@MichMich any idea what your idea behind the hide here was?

Reply:
is this ready to merge or do you need this answer before?

Reply:
@MichMich answer isnt needed for merging, we can clean that up later (hence the TODO)